crates/dreammaker/src/config.rs

```diff
@@ -120,6 +120,10 @@ impl Config {
         Ok(toml::from_str(&config_toml)?)
     }
 
+    pub fn read_toml_string(toml: String) -> Result<Config, Error> {
+        Ok(toml::from_str(&toml)?)
```

Comment:
If this takes `&str`, presumably so could the chain that calls it, rather than requiring `config.to_string()` in the tests